bin/xml/ts.top.import.xml

```diff
@@ -416,7 +416,7 @@
         <property name="prior.status" value="pass,fail,error,notRun"/>
     </target>
     
-    <!-- for default run, pass "!rmi_iiop&!corba" to disable RMI-IIOP/CORBA -->
+    <!-- for default run, pass "!rmiiiop&!corba" to disable RMI-IIOP/CORBA -->
```

Comment:
Should this comment read `"rmiiiop&corba"` instead?

Note that https://www.eclipse.org/lists/jakartaee-tck-dev/msg01050.html is still ongoing and may impact this pr.

Reply:
The PR addresses glassfish runs only.

Reply:
Should the comment be changed to:
```
<!-- for default run, pass "rmiiiop&corba" to disable RMI-IIOP/CORBA -->
```

Reply:
shall we change comment as:
`    <!-- for non glassfish default run, we pass "!rmi_iiop&!corba" to disable RMI-IIOP/CORBA -->
`

Reply:
Small edit to above( if this is what was intended).

<!-- for non-glassfish default run, we set keywords as "!rmi_iiop&!corba" to disable RMI-IIOP/CORBA -->

Reply:
done